plugin-server/tests/worker/vm/upgrades/historical-export/export-historical-events-v2.test.ts

```diff
@@ -583,6 +593,34 @@ describe('addHistoricalEventsExportCapabilityV2()', () => {
                 )
                 expect(await storage().get(EXPORT_PARAMETERS_KEY, null)).toEqual(null)
             })
+
+            it('stops export if abortMessage is set', async () => {
+                await storage().set(EXPORT_PARAMETERS_KEY, { ...params, abortMessage: 'test aborting' })
+
+                await coordinateHistoricalExport({
```

Comment:
Can we call coordinateHistoricalExport again to verify that it doesn't try to stop the export again but rather just hits the shortcircuit for when EXPORT_PARAMETERS_KEY is null?

Reply:
we have a test already for this case `does nothing if export isnt running / is done'`